### ISSUE TITLE
Production and Industrial Facilities - Validation fails with Conformance Class 'INSPIRE GML application schemas' (basic test)

### ISSUE THREAD

**Issue by AurelijaStu** [INSPIRE-MIF/helpdesk-validator#519]
Hello,

I would like to ask for advice about the target schema which is appropriate for data transformation of the **47.01 IED / LCP emission dataset**.
According to the Production and industrial facilities specification, the data should be transformed according to the **ProductionAndIndustrialFacilities GML Application Schema (version: 3)** and/or the **ProductionAnIndustrialFacilitiesExtension Model GML Application Schema (version: 3)**.
In this case, The ProductionAndIndustrialFacilities GML Application Schema (version: 3 or 4) is not suitable for us to transform LCP emission data because some necessary feature types are not included in this schema (for example, ProductionInput, ProductionOutput).
For this reason, I used **http://inspire.ec.europa.eu/draft-schemas/pf-core/2.0/ProductionAndIndustrialFacilitiesCore.xsd** schema which suits to transform the LCP emission data, because important feature types for showing emission data (ProductionInput, ProductionOutput, ProductionProcess) are included in this schema. 
But the problem I face is that the ETF validator returns one error I can't solve: 
„The XML documents representing the spatial data set do not contain a feature of any of the spatial object types in the 'Production and Industrial Facilities' application schemas. Therefore, the spatial data set cannot conform to this conformance class. If you have expected to find spatial objects from the application schemas in the data set, please consult the statistics information to see the spatial object types that have been found.“
I did add spatial object types of the 'Production and Industrial Facilities' application schema, in order to show it, I attached an example of the GML file itself and ETF report to this description. 
I would be grateful if you could tell me what transformation mistake I make that there is no way I can solve this problem.
![validator_report_error](https://user-images.githubusercontent.com/79690432/109277340-e074c380-781f-11eb-86a5-7267ad3cd08c.PNG)
[Validator_report_and_gml_file.zip](https://github.com/INSPIRE-MIF/helpdesk-validator/files/6048473/Validator_report_and_gml_file.zip)

Aurelija

**Comment by carlospzurita** [comment]
Dear @AurelijaStu 

The test for PF was developed using the schema located at https://inspire.ec.europa.eu/schemas/pf/4.0/ProductionAndIndustrialFacilities.xsd , and so the test is failing correctly for this dataset as the schema declared is not supported.

We are marking this as discussion for now, as it seem is more a problem of schema versions rather than an issue on the validator. We will offer some more insight on this after an internal discussion.

Best.

**Comment by AurelijaStu** [comment]
thank you, I'll be waiting for the insights.

**Comment by fabiovin** [comment]
Dear @AurelijaStu,

you are using an old draft schema, so the validator cannot support the validation against it.

The official schemas are those available in this repository: https://inspire.ec.europa.eu/schemas, and in particular only version 4 must be used.

The Application schema _Production and Industrial Facilities Extension_ is only an example of a possible extension of the official application schema.

Please note that the validator uses the schema/s declared in the _schemLocation_ attribute of the GML, this was the reason why only one error was detected in your dataset, so this could lead to considering that the dataset is almost 100% INSPIRE compliant but this is not correct because an unofficial schema is used.

**Comment by AurelijaStu** [comment]
@fabiovin ,
I used the https://inspire.ec.europa.eu/schemas/pf/4.0/ProductionAndIndustrialFacilities.xsd schema in other datasets of the theme „Production and Industrial facilities“. But I don‘t know, how can I use this schema for priority dataset 47,01 IED/LCP emission. I think the main information in this priority dataset is emission data, but I can't find where I must put this information correctly. I think the correct way to show emissions is to fill in the amount, uom, also the name of the emission gas.
I have tried to add and fill the dataTypes InputOutputAmount (amount and InputOutput attributes) and Function (input and output attributes) to the https://inspire.ec.europa.eu/schemas/pf/4.0/ProductionAndIndustrialFacilities.xsd schema (see the attachment). Unfortunately, after data transformation - emission data doesn‘t appear in the GML file.
Perhaps you could advise what I have to do for making the GML file 100% compliant in this case?
![attachment](https://user-images.githubusercontent.com/79690432/110289688-7368e700-7ff2-11eb-9079-c99b28ecba9e.png)

**Comment by fabiovin** [comment]
Dear @AurelijaStu,
since your last question is not strictly related to the validator I suggest you move this discussion to the main helpdesk (https://github.com/INSPIRE-MIF/helpdesk/issues). 

In the meantime, please note that the dataTypes InputOutputAmount is not present in the ProductionAndIndustrialFacilities.xsd, could you verify what xsd you imported?

**Comment by AurelijaStu** [comment]
@fabiovin, 
Yes, you are right. A datatype InputOutputAmount is not present in the ProductionAndIndustrialFacilities.xsd. This dataType we can add in Hale studio when we add target schema ProductionAndIndustrialFacilities.xsd (function in hale studio: „edit mapping relevant target types“). So I can add dataTypes (and featureTypes) from http://inspire.ec.europa.eu/schemas/act-core/4.0/ActivityComplex_Core.xsd schema). 
I tried to provide emission data in various ways. First, I tried old schemes, secondly, I added additional  dataTypes to the latest PF scheme, but in the end, emission information was not moved to the GML file (in case of additional dataType InputOutputAmount) or the validator dropped errors (in the case of using of the old schemes). From your answers, I understood I can't use old schemes, even if they seem suitable for emissions data to show ( as I mentioned earlier, I think it is important information for 47,01 Priority dataset). As well I understood that it is not a validator issue, so I should ask for clarification of the schemes. Did I understand right?
